src/pages/PrimeCalculator/Form/index.tsx

```diff
@@ -416,6 +400,9 @@ export const Form: React.FC = () => {
             control={control}
             render={({ field }) => (
               <Field
+                // This value is incorrect: it should correspond to the amount of tokens taken in
+                // consideration when calculating the Prime rewards. It corresponds to the amount of
+                // tokens supplied by the user, up to the cap defined by Prime
```

Comment:
The value returned by primeEstimationData.supplyCapTokens should top to its cap according to the amount of XVS staked in the estimation